src/Tickets/Emails/Admin/Emails_Tab.php

```diff
@@ -30,6 +31,51 @@ class Emails_Tab {
 	 * @var string
 	 */
 	public static $slug = 'emails';
+	
+	/**
+	 * Holder for template object.
+	 *
+	 * @since TBD
+	 *
+	 * @var null|Tribe_Template
+	 */
+	protected $template;
+
+	/**
+	 * Edit section slug.
+	 * 
+	 * @since TBD
+	 * 
+	 * @var string
+	 */
+	public static $edit_section_slug = 'edit_email';
+
+	/**
+	 * Editing id key.
+	 * 
+	 * @since TBD
+	 * 
+	 * @var string
+	 */
+	public static $key_edit_id = 'id';
+
+	/**
+	 * Key to determine current section.
+	 *
+	 * @since TBD
+	 *
+	 * @var string
+	 */
+	public static $key_current_section = 'tec_tickets_emails_current_section';
+
+	/**
+	 * Key to determine current email id.
+	 *
+	 * @since TBD
+	 *
+	 * @var string
+	 */
+	public static $key_current_email_id = 'tec_tickets_emails_current_email_id';
```

Comment:
The current email id will be the `section` -> something like `admin.php?page=tec-tickets-settings&tab=event-tickets&section=THE_EMAIL_ID` will be the section for each separate email.